code/game/objects/items/weapons/power_cells.dm

```diff
@@ -45,6 +45,11 @@
 	. = ..()
 	charge = 0
 
+/obj/item/cell/crap/cig
```

Comment:
i reckon it'd be better to just use `/obj/item/cell/crap` instead of adding another subtype, just rename that one to `\improper rechargable mini-battery` cuz i doubt we have AA batteries in the future (i hope)

Reply:
As explained on discord and posted here for clarity: the base /obj/item/cell/crap is used too for the deluxe ecigs so I need both the subtype and this one.